app/src/main/res/layout/list_users_adapter.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
A fragment? If I'm not wrong, this layout is use for the RecyclerView's Adapter. Not for inflating fragments.